processing/src/test/java/org/apache/druid/query/topn/TopNQueryRunnerBenchmark.java

```diff
@@ -46,6 +46,7 @@
 /**
  * Based on TopNQueryRunnerTest
  */
+@Ignore
```

Comment:
@Akshat-Jain @kgyrtkirk I was curious why this benchmark was ignored in this patch. Was it related to the switch to jdk 17 that caused the test to start failing? I was trying to run it locally and noticed some memory leaks from the "stupid pool" (which showed up in jdk 17 runs but not with jdk 11?), so I just wanted to check that a legitimate issue wasn’t being masked here

Reply:
* this seems like a benchmark and not a test
* its misplaced as its in processing and not in benchmarks
* iirc there are other ignored benchmarks around here...so addoing it for this one was just following an existing pattern
* I don't fully recall why it come into sight; but migrating it to benchmarks was considered...but it was not that simple
* you could probably narrow down and fix the issue it may have found